pkg/minikube/update/update_test.go

```diff
@@ -1,183 +0,0 @@
-/*
```

Comment:
@hferentschik May be we should keep the file for now and then when we decide what we want to do with update it will be automatically taken care of.